CMSgov/mdct-mcr
# upgrade AdminBanner to use useContext and add related tests

## Description

Moving to use `useContext` for the admin banner, + tests.

### How to test

Nothing should change, so just:
1. Pull and run
2. Login as admin and nav to /admin.
3. Mess with the admin banner form and it should all still work.
4. See automated tests pass.

### Changed Dependencies

No changed deps.

## Code author checklist

## Reviewer checklist (two different people)
